pkg/issuer/acme/dns/util/wait.go

```diff
@@ -199,7 +199,7 @@ func DNSQuery(ctx context.Context, fqdn string, rtype uint16, nameservers []stri
 			// Try TCP if UDP fails
 			if (in != nil && in.Truncated) ||
 				(err != nil && strings.HasPrefix(err.Error(), "read udp") && strings.HasSuffix(err.Error(), "i/o timeout")) {
-				logf.V(logf.DebugLevel).Infof("UDP dns lookup failed, retrying with TCP: %v", err)
+				logf.FromContext(ctx).V(logf.DebugLevel).Info("UDP dns lookup failed, retrying with TCP", "err", err)
```

Comment:
```suggestion
				logf.FromContext(ctx).V(logf.DebugLevel).Error(err, "UDP dns lookup failed, retrying with TCP")
```

Or would that be a change in behavior?

Reply:
I think this will change the behavior, https://pkg.go.dev/github.com/go-logr/logr#LogSink has different functions for `Info` and `Error`, a log sink implementation can implement different logic for both type of calls (eg. log errors to stderr instead of stdout).

Reply:
Ok, let's be conservative here.